kernel/dev/rtc/rtc.cpp

```diff
@@ -62,7 +62,7 @@ uint8_t px_rtc_get_register(int reg) {
 void px_read_rtc() {
     // Previous values from RTC
     // Used as a cache to check if we should update
-    uint8_t century, last_second, last_minute, 
+    uint8_t px_rtc_century, last_second, last_minute, 
```

Comment:
Are these changes related to adding those flags?

Reply:
Tangentially. I had to update the century RTC code and this wasn't named according to the style guide, so I updated it.